src/hwdb/hwdb.c

```diff
@@ -684,18 +693,23 @@ static int hwdb_update(int argc, char *argv[], void *userdata) {
                 return -ENOMEM;
 
         mkdir_parents_label(hwdb_bin, 0755);
-        r = trie_store(trie, hwdb_bin);
-        if (r < 0)
-                return log_error_errno(r, "Failure writing database %s: %m", hwdb_bin);
+        err = trie_store(trie, hwdb_bin);
+        if (err < 0)
+                return log_error_errno(err, "Failure writing database %s: %m", hwdb_bin);
+
+        err = label_fix(hwdb_bin, false, false);
+        if (err < 0)
+                return err;
 
-        return label_fix(hwdb_bin, false, false);
+        return r;
 }
 
 static void help(void) {
         printf("Usage: %s OPTIONS COMMAND\n\n"
                "Update or query the hardware database.\n\n"
                "  -h --help            Show this help\n"
                "     --version         Show package version\n"
+               "  -s --strict          When updating, return non-zero exit value on any parsing error\n"
```

Comment:
Oh, somehow this fell off my radar. Sorry for the delay.

--help output needs to be limited to 80 columns. We actually have a test for this, which should have caught this (this line is 86 characters wide now).

Reply:
Oh, all the CI is failing because of the width issue:
```
252/257 check-help-systemd-hwdb                 FAIL     0.02 s
```

Reply:
Fixed. Thanks.